src/utils/helpers.ts

```diff
@@ -17,30 +17,38 @@ export const generateDownload = (
   URL.revokeObjectURL(url);
 };
 
-// TODO: Make filename timestamp.filetype
-export const downloadPDF = (content: string) => {
+const filename = () => {
+  const title = document.title
+  return title
+}
+
+export const downloadPDF = (content: any) => {
   html2pdf(content, {
     margin: 0,
-    filename: "prompt.pdf",
+    filename: filename() + '.pdf',
```

Comment:
well while adding inline function invocation works, I'd prefer to assign it to a variable and then use it. Also lets keep using template literals so it's consistent

e.g.

```javascript
const name = filename()
...
filename: `${name}.pdf`
```